libraries/classes/DatabaseInterface.php

```diff
@@ -400,6 +401,24 @@ public function getTablesFull(
                 $link
             );
 
+            // here, we check for Mroonga engine and compute the good data_length and index_length
+            // in the StructureController only we need to sum the two values as the other engines
+            foreach ($tables as $one_database_name => $one_database_tables) {
+                foreach ($one_database_tables as $one_table_name => $one_table_data) {
+                    if ($one_table_data['Engine'] !== 'Mroonga' || ! $this->hasMroongaEngine()) {
+                        continue;
+                    }
```

Comment:
```suggestion
                    if ($one_table_data['Engine'] !== 'Mroonga') {
                        continue;
                    }
                    if (! $this->hasMroongaEngine()) {
                        continue;
                    }
```
This would be optimal and avoid wasting a query that would fail anyway, right ?

Reply:
Done